hack/make-rules/vet.sh

```diff
@@ -25,8 +25,14 @@ cd "${KUBE_ROOT}"
 
 # This is required before we run govet for the results to be correct.
```

Comment:
this comment probably needs to be updated - "This" no longer refers to anything.